generic/tests/monotonic_time.py

```diff
@@ -0,0 +1,99 @@
+import os
+import logging
+
+from inspect import ismethod
+
+from virttest import data_dir
+from virttest import error_context
+
+
+class TimeClientTest(object):
+    def __init__(self, test, params, env, test_name):
+        self.test = test
+        self.vm_params = params
+        self.env = env
+        self.vm = self.env.get_vm(self.vm_params["main_vm"])
```

Comment:
If `self.env` and `self.vm_params` is not use anywhere else, maybe you can save the parameter for `self`, `env.get_vm(params["main_vm"])`